.github/workflows/build.yml

```diff
@@ -23,7 +23,13 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -e ".[dev,bnb]"
+        pip install torch
```

Comment:
This would get later overwritten by
```
torch==2.0.0+cu117
```
from requirements.txt

Build log:
```
Collecting torch==2.0.0+cu117
  Downloading https://download.pytorch.org/whl/cu117/torch-2.0.0%2Bcu117-cp38-cp38-linux_x86_64.whl (1843.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.8/1.8 GB 699.2 kB/s eta 0:00:00
```

Reply:
Oh huh. Running `pip install -r requirements.txt` locally I get:
```
ERROR: Could not find a version that satisfies the requirement torch==2.0.0+cu117 (from versions: 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0)
ERROR: No matching distribution found for torch==2.0.0+cu117
```
I guess the CI hosts have CUDA/GPU support? I'll remove the lone `pip install torch`!